scripts/file_uploader.py

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env python
+"""
+Commandline script to upload any file to KE-chain
+"""
+from __future__ import print_function
+
+import logging
+import os
+import sys
+
+from envparse import Env
+
+try:
+    from pykechain import Client
+except ImportError:
+    sys.path.append('..')
+    from pykechain import Client
+
+from pykechain.models import AttachmentProperty
+from pykechain.exceptions import NotFoundError, MultipleFoundError, LoginRequiredError
+
+env = Env()
+env.read_envfile('../.env')
```

Comment:
I think it would be good to unify the environment interaction with pykechain. See pykechain/client.py:51 with `Client.from_env()` and of course the test suite tests/utils.py:7